app/views/posts/index.html.erb

```diff
@@ -1 +1,33 @@
-<h1>Here is a list of all posts by user 1</h1>
\ No newline at end of file
+<article class="main-article" style="width: 90%; margin: auto">
+  <div style="display: flex; align-items: center; justify-content: space-between;">
+    <%= image_tag @user.photo, :class => "user-avatar" %>
+    <div class="user-details">
+      <p><%= @user.name %></p>
+      <span>Number of posts: <%= @user.post_counter %></span>
+    </div>
+  </div>
+
+  <% @user.recent_posts.each_with_index do |post, index| %>
+    <div>
+      <div class="post">
+        <h4>Post #<%= index + 1 %></h4>
+        <div>
+          <p class="post-title"><%= post.title %></p>
```

Comment:
- [ ] [Optional] Well! you are showing the project title here. Considering that the title might be longer and the description is typically shorter in font size, I recommend displaying the post description here instead of the title for a more visually balanced and informative representation.

Reply:
@shahadat3669 There is no field named description on the post.

Reply:
@simplegoose Sorry, it should be `text`.